sdk/search/search-documents/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/search-documents",
-  "version": "1.0.0-preview.5",
+  "version": "11.0.0",
```

Comment:
Should this have a preview.5 still? I am not sure if the release pipeline in JS will take care of removing the suffix or not.

Reply:
@AlexGhiondea It we have preview then it will be released as `11.0.0-preview`. I do not think we need that. We want to release this as `11.0.0` and mention the same. The release pipeline does not do any changes.